File: src/Features.FSharp/DynamicModel.fs
namespace Features

open FSharp.Interop.Dynamic

type DynamicModel() =
    member this.InvokeJavascriptFunction(funct: obj, message: string) =
        !?funct message |> ignore

    member this.Add(source: obj, destination: obj) =
        destination?computedResult <- source?value1 + source?value2

File: src/Features.CSharp/DynamicModel.cs
namespace Features
{
    public class DynamicModel
    {
        public void InvokeJavascriptFunction(dynamic function, string message)
        {
            function(message);
        }

        public void Add(dynamic source, dynamic destination)
        {
            destination.computedResult = source.value1 + source.value2;
        }
    }
}